### DISCUSSION TITLE
How can we prevent NumberInput from allowing the value to be cleared?

### DISCUSSION THREAD

**Discussion by Nantris** [mantinedev/mantine#3224] [Q&A]
I'd like instead for the value to revert to the last valid value instead. It's already a controlled input, but the value isn't honored in this exact situation.

**Comment by rtivital** [comment]
It is the same issue as https://github.com/mantinedev/mantine/issues/3059

**Reply by Nantris** [reply]
I don't think so, because the value stays blank even when the input is blurred.

**Reply by Nantris** [reply]
Any thoughts about this? It's problematic in our use case that the field can be blanked out, which isn't the case if we use a plain `<input />` instead of the Mantine-provided one.

**Reply by rtivital** [reply]
The issue is not solvable with current implementation, as I've pointed out before. I'm researching the option to migrate the value to string to avoid this issue.

**Reply by Nantris** [reply]
@rtivital sorry I misunderstood, I thought you had written "_Is_ it the same issue" not "It _is_". Thanks for your great work!
